core/types/block.go

```diff
@@ -33,8 +33,8 @@ import (
 )
 
 var (
-	OldBlockMillisecondsInterval uint64 = 1000
-	NewBlockMillisecondsInterval uint64 = 500
+	DefaultBlockIntervalUintCount uint64 = 2
```

Comment:
DefaultBlockIntervalUintCount may need change to 4 if the BlockMillisecondsIntervalUint change to 250ms